plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/ContentTypeService.java

```diff
@@ -179,9 +179,34 @@ public Integer getComponentUsage(String componentCode) {
                 .map(contentDto -> new ComponentUsageEntity(
                         ComponentUsageEntity.TYPE_CONTENT,
                         contentDto.getId(),
-                        contentDto.getStatus().equals(Content.STATUS_READY) ? IPageService.STATUS_ONLINE : IPageService.STATUS_DRAFT))
+                        getContentStatus(contentDto)))
                 .collect(Collectors.toList());
 
         return pagedMetadataMapper.getPagedResult(restListRequest, componentUsageEntityList, "code", pagedData.getTotalItems());
     }
-}
+
+
+    /**
+     * calcs and returns the string representation of a Content status
+     * @param contentDto the ContentDto of which return the status representation
+     * @return the string representation of a Content status
+     */
+    private String getContentStatus(ContentDto contentDto) {
+
+        if (contentDto.getStatus().equals(Content.STATUS_PUBLIC)) {
+            return "Published";
+        } else if (contentDto.getStatus().equals(Content.STATUS_READY)) {
+            if (contentDto.isOnLine()) {
+                return Content.STATUS_PUBLIC + " ≠ " + Content.STATUS_READY;
+            } else {
+                return Content.STATUS_READY;
+            }
+        } else {
+            if (contentDto.isOnLine()) {
+                return Content.STATUS_PUBLIC + " ≠ " + Content.STATUS_DRAFT;
```

Comment:
What are we using this string for? Are we going to run into encoding problems with "≠" ? Seems like we might be better off with a standard string, especially if this is coming back in an API result.